subsys/fs/nvs/nvs.c

```diff
@@ -715,6 +741,34 @@ static int nvs_startup(struct nvs_fs *fs)
 	}
 	if (rc) {
 		/* the sector after fs->ate_wrt is not empty */
+		bool gc_done_marker = false;
+		struct nvs_ate gc_done_ate;
+
+		addr = fs->ate_wra + ate_size;
```

Comment:
maybe a comment for emphasizing that following loop will scan the ATE area of the open sector (selected before) for GC DONE ATE

Reply:
done